app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb

```diff
@@ -0,0 +1,73 @@
+class ManageIQ::Providers::IbmCloud::Inventory::Parser::VPC < ManageIQ::Providers::IbmCloud::Inventory::Parser
+  require_nested :CloudManager
+
+  attr_reader :img_to_os
+
+  def initialize
+    @img_to_os = {}
+  end
+
+  def parse
+    images
+    instances
+  end
+
+  def images
+    collector.images.each do |image|
+      img_to_os[image[:id]] = image&.dig(:operating_system, :name)
```

Comment:
If we do this in the collector it'll be a little easier to work with when we do targeted refresh (because we can control the targets from there unlike the parser).

e.g.
```
class Collector
  def images
    @images ||= collector.images.all
  end

  def operating_system_by_image_id
    @operating_system_by_image_id ||= images.each_with_object({}) { |img, h| h[img[:id]] = img.dig(:operating_system, :name) }
  end
end
```